.github/workflows/test_build_and_deploy.yml

```diff
@@ -44,9 +44,10 @@ jobs:
       - name: GitOps (build, push and deploy a new Docker image)
         uses: ./.github/gitops
         with:
+          dockerregistry: index.docker.io
           dockerusername: ${{ secrets.DOCKER_USERNAME }}
           dockerpassword: ${{ secrets.DOCKER_PASSWORD }}
-          dockerimage: public/syseleven-exporter
+          dockerimage: staffbace/syseleven-exporter
```

Comment:
```suggestion
          dockerimage: staffbase/syseleven-exporter
```

Reply:
This is the correct organization, `staffbase` was already taken.